lapce-ui/src/about.rs

```diff
@@ -446,9 +442,16 @@ impl Widget<LapceTabData> for AboutBoxContent {
         .with_origin(Point::new(
             self.width / 2.0 + (row_item.layout.width() as f64 / 2.0)
                 - row_item.layout.width() as f64,
-            rect.y1 - row_item.layout.height() as f64 * 2.0,
+            y,
         ));
 
+        ctx.fill(
+            site_rect,
+            &data.config.get_hover_color(
+                data.config.get_color_unchecked(LapceTheme::TERMINAL_RED),
```

Comment:
It's not terminal so don't use that

Reply:
LOL I left it over while debugging. My bad.